Makefile

```diff
@@ -55,7 +55,7 @@ NETGO_CHECK=@strings $@ | grep cgo_stub\\\.go >/dev/null || { \
 	echo "    sudo go install -tags netgo std"; \
 	false; \
 }
-BUILD_FLAGS=-i -ldflags "-extldflags \"-static\" -X main.version=$(WEAVE_VERSION)" -tags netgo
+BUILD_FLAGS=-i -ldflags "-linkmode external -extldflags -static -X main.version=$(WEAVE_VERSION)" -tags netgo
```

Comment:
There is more to it - without this flag Go uses the external linker for all our programs except `kube-peers`.  The logic to decide which linker to use is rather convoluted - I reckon it was accidentally working from day one.